config.md

```diff
@@ -56,14 +56,14 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
   This value MUST be an absolute path.
   * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
   * Solaris: corresponds to "dir" of the fs resource in [zonecfg(1M)][zonecfg.1m].
-* **`type`** (string, OPTIONAL) The filesystem type of the filesystem to be mounted.
-  * Linux: valid *filesystemtype* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
+* **`type`** (string, OPTIONAL) The type of the filesystem to be mounted.
+  * Linux: the types of filesystem supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
```

Comment:
> I used `types` not `type`.

But why use “type(s)” at all?  I still prefer the more compact form from [here][1], but if you want to use “type(s)” I'd recommend either:

> Linux: filesystem types supported by the kernel are listed in `/proc/filesystems` (e.g., `minix`, `ext2`, `ext3`, …).

or:

> Linux: types of filesystems supported by the kernel are listed in `/proc/filesystems` (e.g., `minix`, `ext2`, `ext3`, …).

[1]: https://github.com/opencontainers/runtime-spec/pull/807#discussion_r116557329